### PR TITLE
Moved FormikDebug to be inside packages/forms

### PR DESCRIPTION
**Pull request by paulstroot**
This component is used internally by lib/FormikWrapper.

### CODE REVIEW

File: lib/index.js
Comment (cdnicoll) [inline lib/index.js:1]: Can we convert this to .ts

File: lib/FormikWrapper.jsx
Comment (cdnicoll) [inline lib/FormikWrapper.jsx:1]: and this one to .tsx

**Review by cdnicoll — COMMENTED** [review]
Thanks @paulstroot just some conversion from js to ts. Thanks!

**Comment by paulstroot** [comment]
Conversion to tsx complete @cdnicoll , I used chatGPT for a lot of it, so hopefully all looks correct. You might want to give it a quick glance over. The one thing that I was getting a console warning about was the use of the type "any" in a few places.

**Review by cdnicoll — APPROVED** [review]

**Pull request merged**